src/pyglottolog/languoids/models.py

```diff
@@ -186,7 +186,31 @@ def _format(self, minimal=True):
 
     @classmethod
     def from_name(cls, name):
+        # some additional common names
+        _code_map = {
```

Comment:
Can we be more explicit about where these are missing (`pycountry`?).

nit: If we do want to locally extent the countries, this should better go into a module-level constant.